content/docs/authorization/_index.md

```diff
@@ -68,6 +68,7 @@ CSM for Authorization consists of 2 components - the Authorization sidecar and t
 | Authorization Sidecar Image Tag | Authorization Proxy Server Version |
 | ------------------------------- | ---------------------------------- |
 | dellemc/csm-authorization-sidecar:v1.0.0 | v1.0.0, v1.1.0 |
+| dellemc/csm-authorization-sidecar:v1.2.0 | v1.0.0, v1.1.0, v1.2.0 |
```

Comment:
Please remove v1.0.0 from this new entry

Reply:
Addressed